Sources/BraintreeSEPADirectDebit/SEPADebitAccountsRequest.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+/// The POST body for `v1/payment_methods/sepa_debit_accounts`
+struct SEPADebitAccountsRequest: Encodable {
+
+    private let createMandateResult: SEPADebitAccountRequest
+
+    enum CodingKeys: String, CodingKey {
+        case createMandateResult = "sepa_debit_account"
+    }
+
+    init(createMandateResult: CreateMandateResult) {
```

Comment:
Nit - I would move the init to the bottom of this class.

Maybe these new Encodable structs is a good candidate for us to add to our style guide how we should organize these, MARKs, etc

Reply:
I had followed the pattern we established in [BTApplePaymentTokensRequest](https://github.com/braintree/braintree_ios/blob/main/Sources/BraintreeApplePay/BTApplePaymentTokensRequest.swift) where the nested struct is under the `init`. Though I do think having structs within structs could be making our preferred style more convoluted. Thoughts on removing the nesting?

Reply:
> Yeh, looking back I think having all the struct definitions be near the property that uses it & pushing the init down is more readable vs having them intertwined. What do you think?

Yeah I am down if we feel like it makes things more readable! I pushed a commit here with the change: https://github.com/braintree/braintree_ios/pull/1173/commits/d479ab91ac3697cc7b8c32461d0250eb416470ab

Also agree on style guiding whatever we decide (and finding a home for the style guide).

> In terms of nested structs, I think they sometimes help readability to know when something is defined within the context of something else. For example with the ApplePaymentToken one, that name is pretty generic, but namespacing it the <Request>.ApplePaymentToken helps reduce name collisions.

That makes sense with naming collisions, I forgot about that aspect. Especially with something like `card` that could be a situation where a merchant has something similar by the same name.

Reply:
I added a backlog ticket for updating our style guide - DTBTSDK-3373